Doc/library/signal.rst

```diff
@@ -503,3 +503,32 @@ be sent, and the handler raises an exception. ::
 
    signal.alarm(0)          # Disable the alarm
 
+Note on SIGPIPE
```

Comment:
By the way, I don't think this is suitable for the signal module documentation. Maybe we can create a HOWTO document for signals? A HOWTO document would also allow us adding more best practices in the future.

Reply:
Agreed. Until we have such document, I'm fine putting this in the documentation as is so it's searchable.